docs/tools/vdb_table/vendor.schema.json

```diff
@@ -111,6 +107,17 @@
         "comment": {"type":"string"}
       }
     },
+    "statsWithSource": {
+      "$id": "statsWithSource",
+      "type": "object",
+      "properties": {
+        "value": {"type": "integer"},
+        "value_90_days": {"type": "integer"},
+        "unlimited": {"type": "boolean"},
```

Comment:
@chirag-jn is unlimited still needed here?

Reply:
I am not sure about its usage. After discussing with @AruneshSingh I got to know that we can remove this field. Starting with that now.